.github/workflows/release-sigstore-java-from-tag.yaml

```diff
@@ -56,22 +48,30 @@ jobs:
           java-version: 11
           distribution: 'temurin'
 
-      - name: Build project
+      - name: Build, Sign and Release to Maven Central
         run: |
-          ./gradlew clean :sigstore-java:createReleaseBundle -Pversion=${{ github.event.inputs.release_version }} -Prelease -PskipSign
+          ./gradlew clean :sigstore-java:publishMavenJavaPublicationToSonatypeRepository -Prelease
+        env:
+          ORG_GRADLE_PROJECT_signingKey: ${{ secrets.PGP_PRIVATE_KEY }}
+          ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.PGP_PASSPHRASE }}
+          ORG_GRADLE_PROJECT_sonatypeUsername: ${{ secrets.SONATYPE_USERNAME }}
+          ORG_GRADLE_PROJECT_sonatypePassword: ${{ secrets.SONATYPE_PASSWORD }}
```

Comment:
A generic questions. I wonder, if it is "safer" to use system environment or `-P` in the Gradle calls?

System envs are populated across the system/container. On the other hand, `-P` params might be exposed on the process list.

Reply:
Yeah, I mean that's probably true. I can use `-PsigningKey`, etc here instead

Reply:
oh wait, I don't think github actions allows you to access secrets directly.

Reply:
You might be right. I'm not sure if `-PsigningKey=${{ secrets.PGP_PRIVATE_KEY }}` is supported. Even if, in the documentation, using envs is recommended, so probably their try to protect secrets at the "container" level (and keeping it as is might be a better idea).